sim/dex.ts

```diff
@@ -314,6 +314,24 @@ class ModdedDex {
 		}
 	}
 
+	getEffectivenessCode(move: Move, target: Pokemon): number {
```

Comment:
I'm not sure if a function taking a `Pokemon` belongs in `dex.ts`...

Reply:
On hindsight, what I meant is ``Template``...